flaskr/templates/blog/index.html

```diff
@@ -1,9 +1,9 @@
 {% extends 'base.html' %}
 
 {% block header %}
-  <h1>{% block title %}Posts{% endblock %}</h1>
+  <h1>{% block title %}{{ get_translation(locale, 'Posts') }}{% endblock %}</h1>
```

Comment:
execution has the following error
jinja2.exceptions.UndefinedError: 'get_translation' is undefined